package.json

```diff
@@ -139,6 +141,7 @@
     "karma-super-dots-reporter": "0.2.0",
     "lazypipe": "1.0.1",
     "lolex": "3.0.0",
+    "magic-string": "^0.25.1",
```

Comment:
Let's not use the hat version here, can we specify the exact pinned version (`0.25.1`).

Reply:
done